src/main/resources/liquibase/harness/change/expectedSql/postgresql/dropAllForeignKeysMixedCaseTablesWithProKey.sql

```diff
@@ -0,0 +1,2 @@
+CREATE TABLE public."MixedCase1" (id INTEGER NOT NULL, CONSTRAINT "MixedCase1_pkey" PRIMARY KEY (id))
+CREATE TABLE public."MixedCase2" (id INTEGER, ref_id INTEGER, CONSTRAINT fk_mixed FOREIGN KEY (ref_id) REFERENCES public."MixedCase1"(id))
```

Comment:
There's a bug with this change type. @yodzhubeiskyi just FYI. The 'DROP FOREIGN KEY' statement doesn't get generated.

Reply:
@yodzhubeiskyi -- please follow up on this tomorrow.

Reply:
Follow up from @yodzhubeiskyi --
>Hi @kristyl, good morning. So the PR itself is fine. The bug, that @Oleksandr Sashchenko mentioned is related to current version of test-harness and not PR itself. However, it doesn't affect correct verification of the test.

@AlexanderSashchenko -- Let's follow up on that bug and I also see some tests failing that not postgres related.